packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -688,6 +695,12 @@ export default class extends Component {
                       ))}
                   </td>
                 </tr>
+                {mdadmHealthAvailable && (
+                  <tr>
+                    <th>{_('raidStatus')}</th>
+                    <td>{mdadmState === 'clean' || mdadmState === 'active' ? _('raidHealthy') : mdadmState}</td>
+                  </tr>
+                )}
```

Comment:
Avoid to hide information, add a message that explain "Why mdadmHealthAvailable is false". 
It will probably depend of the `mdadmHealth`, but it can be `plugin not installed`, or `no raid on the host`